geode-core/src/main/java/org/apache/geode/internal/cache/DistributedRegion.java

```diff
@@ -1034,6 +1034,29 @@ public void initialize(InputStream snapshotInputStream, InternalDistributedMembe
     }
 
     try {
+      // Validation here.
+      Set<String> allGatewaySenderIds = getAllGatewaySenderIds();
+
+      if (!allGatewaySenderIds.isEmpty()) {
+        for (GatewaySender sender : this.cache.getAllGatewaySenders()) {
+          if (sender.isParallel() && allGatewaySenderIds.contains(sender.getId())) {
+            // Fix for Bug#51491. Once decided to support this configuration we have call
```

Comment:
This bug number is a holdover from before Geode was opensourced. Please use an appropriate description of the bug rather than an obsolete tracking number. Yes, it's an old comment, but now would be a good time to clean up the comment since you're touching this part of the code.